year_2021/pyproject.toml

```diff
@@ -5,18 +5,18 @@ description = "Training and Code Test Answers"
 authors = ["beerios <chris@harleybrewing.com>"]
 
 [tool.poetry.dependencies]
-python = "~3.8"
-requests = "^2.24.0"
-marshmallow = "^3.7.1"
+python = "~3.11"
+requests = "^2.31.0"
+marshmallow = "^3.20.1"
 
 [tool.poetry.dev-dependencies]
-pytest = "^6.2.5"
-pylint = "^2.12.0"
-flake8 = "^4.0.0"
-wrapt = "^1.12.1"
+pytest = "^7.4.0"
+pyyaml = "^6.0.1"
```

Comment:
Why was this added?

Reply:
A sub dependency was blocking the build. I think this was actually related to the docker-compose and can be removed.

Reply:
I will remove this and update the PR.

Note, I can'e 'require changes' as the owner of the PR.